From 9295a4e5d4aa8344241c76e458fcce726b765aa6 Mon Sep 17 00:00:00 2001
From: Benjamin Herrera Navarro <bherreranavarro@winsystems.com>
Date: Wed, 1 Nov 2023 17:06:47 +0000
Subject: [PATCH 1/1] Add correct C444 DTB

Signed-off-by: Benjamin Herrera Navarro <bherreranavarro@winsystems.com>
---
 iMX8M/soc.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index a98195e..c93002e 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -142,7 +142,7 @@ clean:
 
 dtb = evk.dtb
 $(dtb):
-	./$(DTB_PREPROC) $(PLAT)-evk.dtb $(dtb) $(dtbs)
+	./$(DTB_PREPROC) $(PLAT)-itx-p-c444.dtb $(dtb) $(dtbs)
 
 u-boot.itb: $(dtb)
 	./$(PAD_IMAGE) $(TEE)
-- 
2.25.1

